04_setup_ironic.sh

```diff
@@ -53,6 +53,11 @@ if [ ! -e "$IRONIC_DATA_DIR/html/images/$RHCOS_IMAGE_FILENAME_DUALDHCP" ] ; then
     sudo kpartx -d /dev/${LOOPBACK}
     sudo losetup -d /dev/${LOOPBACK}
     qemu-img convert -O qcow2 -c "$RHCOS_IMAGE_FILENAME_RAW" "$RHCOS_IMAGE_FILENAME_DUALDHCP"
+
+    # Reduce the disk size so that ironic doesn't have to write as much data during deploy
+    # This is a safe opperation only because the partitions on the disk are only about 3.3G
+    # TODO: add check for partition size to ensure we don't zap valid data if they change in future
+    qemu-img resize --shrink "$RHCOS_IMAGE_FILENAME_DUALDHCP" 4G
```

Comment:
I'm going to look into making this safer if you want to wait before merging.

Reply:
Yeah maybe we could use virt-df or something to calculate the size?

This works fine though so happy to merge this as-is then you can push a followup to validate/calculate the size?